lib/autoprefixer.d.ts

```diff
@@ -0,0 +1,107 @@
+import { Plugin } from 'postcss'
+import { Stats } from 'browserslist'
+
+declare function autoprefixer<T extends string[]> (
```

Comment:
Why do we need `T`?

Reply:
That is the way to implement leading rest parameter in TypeScript 4. In future it will be looks much better: https://github.com/microsoft/TypeScript/pull/41544